src/components/Button/Button.jsx

```diff
@@ -0,0 +1,9 @@
+function Button() {
+  return (
+    <div>
+      <button type="button" className="bg-[#4D2DB7]"></button>
```

Comment:
if you create a reusable button component you should also add the onClick function to it